front_end/src/components/searchResultOverview/searchResultOverview.js

```diff
@@ -10,13 +10,49 @@ import {
   TablePagination
 } from "@material-ui/core";
 import SearchWidget from "../landing_page/search_widget/SearchWidget";
+import { connect } from "react-redux";
+import { displayResultsOverview } from "../../actions/searchResultActions";
 
 class searchResultOverview extends Component {
   render() {
+    const result = this.props.result.result;
+    let hotels;
+
     const styles = {
       rating: { float: "left", width: "100%" },
       imageStyle: { margin: 20, width: 200, height: 200, float: "left" }
     };
```

Comment:
1
The 'styles' object can be moved outside of the component and injected into it by using 'withStyles(styles)(searchResultOverview)' and using the 'classes' prop that is passed into the component with 'classNames'.

It may seem unnecessary but according to Material-UI, it overrides the CSS injected by Material-UI which by default has the highest specificity. Doing so prevents any CSS injection order issues from occurring. More info here: https://material-ui.com/customization/overrides/

It's also good practice to do, and helps with the code structure as you have less code inside the component definition.

Reply:
Fixed 💃 :)